src/pyscaffold/update.py

```diff
@@ -90,10 +98,43 @@ def add_entrypoints(struct: Structure, opts: ScaffoldOpts) -> "ActionParams":
 
     setupcfg[add_after_sect].add_after.section(new_section).space()
 
-    if not opts["pretend"]:
-        setupcfg.update_file()
+    return setupcfg, opts
 
-    return struct, opts
+
+@_change_setupcfg
+def update_setup_cfg(setupcfg: ConfigUpdater, opts: ScaffoldOpts):
+    """Update `pyscaffold` in setupcfg and ensure some values are there as expected"""
+    setupcfg = read_setupcfg(opts["project_path"])
+    if "options" not in setupcfg:
+        setupcfg["metadata"].add_after.section("options")
+
+    if "pyscaffold" not in setupcfg:
+        setupcfg.add_section("pyscaffold")
+    setupcfg["pyscaffold"]["version"] = pyscaffold_version
+
+    logger.report("updated", opts["project_path"] / SETUP_CFG)
+    return setupcfg, opts
+
+
+@_change_setupcfg
+def add_dependencies(setupcfg: ConfigUpdater, opts: ScaffoldOpts):
+    """Add dependencies"""
+    options = setupcfg["options"]
+    if "install_requires" in options:
+        install_requires = options.get("install_requires", Object(value=""))
+        install_requires = deps.add(deps.RUNTIME, deps.split(install_requires.value))
+        options["install_requires"].set_values(install_requires)
+    else:
+        options.set("install_requires")
+        options["install_requires"].set_values(deps.RUNTIME)
```

Comment:
Always add `import-metadata` backport as a dependency in the generated packages to allow fallback when finding the  version from setuptools_scm. This is controversial, see discussion in the [`__init__.template`](https://github.com/pyscaffold/pyscaffold/pull/318/files#r468882115).

Reply:
Looks good for me. Also here a comment would be nice that this only needs to be done until `python_requires = >= 3.8`

Reply:
Question: are we adding a `python_requires = >= 3.6` by default in the generated `setup.cfg`?